include/links-domain.php

```diff
@@ -99,11 +99,20 @@ public function get_hosts() {
 		$hosts = array();
 		foreach ( $this->options['domains'] as $lang => $domain ) {
 			$host = wp_parse_url( $domain, PHP_URL_HOST );
+
+			if ( ! is_string( $host ) ) {
+				continue;
+			}
+
 			// The function idn_to_ascii() is much faster than the WordPress method.
 			if ( function_exists( 'idn_to_ascii' ) ) {
 				// The use of the constant is mandatory in PHP 7.2 and PHP 7.3 to avoid a deprecated notice.
 				$hosts[ $lang ] = defined( 'INTL_IDNA_VARIANT_UTS46' ) ? idn_to_ascii( $host, 0, INTL_IDNA_VARIANT_UTS46 ) : idn_to_ascii( $host );
+			} elseif ( class_exists( 'WpOrg\Requests\IdnaEncoder' ) ) {
+				// Since WP 6.2.
+				$hosts[ $lang ] = \WpOrg\Requests\IdnaEncoder::encode( $host );
 			} else {
+				// Compat WP < 6.2.
```

Comment:
Just to ease searches:
```suggestion
				// Backward compatibility with WP < 6.2.
```